### ISSUE TITLE
Great site!!

### ISSUE THREAD

**Issue by jennah2121** [fac-13/crypt-os#28]
It looks amazing! I love the animation on the icons. Well done for making the home page responsive too!!  👏👏

**Comment by helenzhou6** [comment]
Yes I love the look of it! Well done guys it's super awesome (and informative) 😍

**Comment by nik0lz** [comment]
Thank you :smiley_cat:
